src/Sections/EntitiesSection/Entities/SeqEnd.ts

```diff
@@ -1,12 +1,14 @@
-import { Dxfier } from 'Internals/Dxfier'
+import { Dxfier } from '../../../Internals/Dxfier'
 import Entity from '../Entity'
 
 export default class SeqEnd extends Entity {
-  constructor() {
+  owner: string|undefined
+  constructor(owner?: string) {
     super('SEQEND')
+    this.owner = owner || undefined
```

Comment:
I am not sure why you need this owner here you can change on the instance directly